uber/models.py

```diff
@@ -576,7 +576,8 @@ def change_badge(self, attendee, badge_type, badge_num=None):
                 return out_of_range
 
             # Keeps non-preassigned badges numberless unless they've already been checked in.
```

Comment:
We previously neglected to set the badge type in this if block.

We now also enter this block if ``c.NUMBERED_BADGES`` isn't set, since at that point there's no reason to do badge shifting.